src/config.ts

```
@@ -0,0 +1,2 @@
+export const API_URL = import.meta.env.VITE_API_URL;
```

ESLint

[error] Delete `;`

(prettier/prettier)